scripts/check-node-version-alignment.js

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { readFile, readdir } from "node:fs/promises";
+import { resolve, dirname } from "node:path";
+import { fileURLToPath } from "node:url";
+
+/**
+ * This script checks whether every place we set a Node version is set to the same version.
+ *
+ * I've opted to avoid adding e.g. TOML and YAML parsers, and just assume
+ * specific line formatting, to write the simplest thing that works in the least
+ * amount of time.
```

Comment:
Do you want to file a followup in the backlog to add these? I agree that it's less urgent, might be a good first bug for someone if you think it's valuable (in which case I'd file it via GitHub which will mirror to Jira)

Reply:
I'd suggest to maybe let's first see whether the lack of them is actually problematic. If not, the extra dependencies might not be worth it.